demisto_sdk/commands/common/content/content.py

```diff
@@ -127,3 +132,185 @@ def content_descriptor(self) -> Optional[ContentDescriptor]:
             descriptor_object = ContentDescriptor(path)
 
         return descriptor_object
+
+    def modified_packs(self, prev_ver='master', committed_only=False, staged_only=False) -> Dict[str, Pack]:
+        content_repo: Repo = self.git()
```

Comment:
perhaps this should be a class attribute or it makes sense to initialize it for every func ?